src/contexts/language/LanguageContext.js

```diff
@@ -0,0 +1,3 @@
+import * as React from "react";
+
+export const LanguageContext = React.createContext(undefined);
```

Comment:
Trochę to inaczej powinno być zaimplementowane. Podeślę później jakieś przykładowe rozwiązanie